pandas/tests/test_resample.py

```diff
@@ -650,6 +650,7 @@ def test_asfreq_fill_value(self):
     def test_resample_interpolate(self):
         # # 12925
         df = self.create_series().to_frame('value')
+        df = df.iloc[0:(len(df) // 2)]
```

Comment:
I'm curious, why this change?

Reply:
oh that test appears as the number 1 duration (like 30 s), locally it took 10 though, so made it 1/2